### ISSUE TITLE
How to configure image quality settings

### ISSUE THREAD

**Issue by samgoud** [novnc/noVNC#1288]
**Describe the bug**
We are using tigervnc 1.9 to launch vnc server with -xstartup to launch a qt gui app
We were comparing the display quality of this app when viewed on novnc vs when the app was directly launched on the remote desktop.
What we noticed is in novnc the image quality is a little distorted in comparison
Below is a comparison:  
![image](https://user-images.githubusercontent.com/32926439/64209077-35e74300-cebd-11e9-9e8f-c91363b564b9.png)

Are there any settings in novnc we need to configure to make the picture better.

**Comment by samhed** [comment]
No, I'm afraid there is no such setting at the moment, see #396 for adding this feature.

Until then you will have to manually modify the code for QualityLevel and CompressLevel here:
https://github.com/novnc/noVNC/blob/master/core/rfb.js#L1230

**Comment by samhed** [comment]
I'd guess what you'd want is to increase the quality level and decrease the compress level. The code om noVNC uses quality level 6 and compress level 2, try increasing quality to 9 (which is max) and turnn of compression by changing the number to 0.

**Comment by samgoud** [comment]
Thanks samhed, 
I will try the suggested change

**Comment by samhed** [comment]
You'd be using more CPU on the client and more bandwidth over the network. Not sure about the memory impact, but I'd guess it wouldn't change much.

**Comment by samgoud** [comment]
We are using a old version of novnc. Not sure which version exactly it is. From package.json I see
  "name": "noVNC",
  "version": "0.6.1",

In the version we have, I could not see the exact variables as suggested by you in rfb.js 

I found similar variables in our rfb.js with same encode values as in the latest noVnc
![image](https://user-images.githubusercontent.com/32926439/65240929-00b73380-db00-11e9-974b-6642278a7289.png)

I modified the settings to below
![image](https://user-images.githubusercontent.com/32926439/65241015-30fed200-db00-11e9-8a63-8a2ca936ce90.png)

image quality looks better now.

Does the modification looks ok ? I have attached the rfb.js file for your reference
[rfb.zip](https://github.com/novnc/noVNC/files/3631006/rfb.zip)
